src/hotspot/share/runtime/deoptimization.hpp

```diff
@@ -56,6 +56,7 @@ class Deoptimization : AllStatic {
     Reason_array_check,           // saw unexpected array class (aastore @bci)
     Reason_intrinsic,             // saw unexpected operand to intrinsic (@bci)
     Reason_bimorphic,             // saw unexpected object class in bimorphic inlining (@bci)
+    Reason_polymorphic,           // saw unexpected object class in polymorphic inlining (@bci)
```

Comment:
please update Deoptimization::_trap_reason_name in deoptimization.cpp

Reply:
done